src/test/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervalsIntegrationTest.java

```diff
@@ -0,0 +1,146 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.Interval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.io.File;
+
+public final class PreprocessIntervalsIntegrationTest extends CommandLineProgramTest {
+    private static final File REFERENCE = new File(b37_reference_20_21);
+
+    @DataProvider(name = "intervalInputsFromCommandLine")
+    public final Object[][] testData() {
```

Comment:
Since the class is `final`, all methods are implicitly `final`, so you can change this here and below.  We usually aren't too careful about declaring methods `final`, in general.

Reply:
OK